third_party/tensorrt/local/BUILD

```diff
@@ -351,11 +351,6 @@ cc_library(
         ":ci_rhel_x86_64_linux": glob(["lib64/libnvinfer_plugin.so"]),
         "//conditions:default": ["lib/x86_64-linux-gnu/libnvinfer_plugin.so"],
     }),
-    hdrs = select({
```

Comment:
Why did this get removed?

Reply:
There was a conflict with the base branch. I resolved it by replacing it with the ci_rhel_x86_64_linux version